plugins/plotly-express/src/PlotlyExpressChartPanel.tsx

```diff
@@ -20,27 +20,72 @@ export interface PlotlyExpressChartPanelProps extends ChartPanelProps {
 function PlotlyExpressChartPanel(props: PlotlyExpressChartPanelProps) {
   const dh = useApi();
   const { fetch, ...rest } = props;
+  const containerRef = useRef<HTMLDivElement>(null);
+  const [model, setModel] = useState<PlotlyExpressChartModel>();
 
   const makeModel = useCallback(async () => {
     const widgetInfo = await fetch();
     const data = getWidgetData(widgetInfo);
     const { plotly, deephaven } = data;
     const isDefaultTemplate = !deephaven.is_user_set_template;
     const tableColumnReplacementMap = await getDataMappings(widgetInfo);
-    return new PlotlyExpressChartModel(
+    const m = new PlotlyExpressChartModel(
       dh,
       tableColumnReplacementMap,
       plotly.data,
       plotly.layout ?? {},
       isDefaultTemplate,
       ChartTheme
     );
+    setModel(m);
+    return m;
   }, [dh, fetch]);
 
+  useEffect(
+    function handle3DTicks() {
+      if (!model || !containerRef.current || !model.has3D()) {
+        return;
+      }
+
+      const container = containerRef.current;
+
+      function handleMouseDown() {
+        model?.pauseUpdates();
+        // The once option removes the listener after it is called
+        window.addEventListener('mouseup', handleMouseUp, { once: true });
```

Comment:
I don't think this actually manifests into a problem, but if the panel is unmounted while your mouse is still down, this listener would still fire and `model?.resumeUpdates` would get called. Should be cancelling this listener on unmount as well.